### PR TITLE
added node_modules to .gitignore && changed checkbox.js

### PR DESCRIPTION
**Pull request by alexmuzenhardt**
I have added the node_modules folder to gitignore.
You can run "npm i" to download dependencies.

changed variable name "f" to "labelAttribute" like in Issue #5 
deleted unnecessary semicolon
changed "==" to "==="

**NOTE:**
I will add a package.json and delete node_modules with another PR.

Please review two files to commit and merge.
Thanks.

### CODE REVIEW

File: library/checkbox/checkbox.js [hunk 1]
```diff
@@ -7,21 +7,21 @@ export class Checkbox {
...
           let state = this.getAttribute("data-value");
-          state == "off" ? this.setAttribute("data-value", "on") : this.setAttribute("data-value", "off");
+          state === "off" ? this.setAttribute("data-value", "on") : this.setAttribute("data-value", "off");
         });
       });
       // Attach event listener to the checkbox label
       document.querySelectorAll(".checkbox-label").forEach(function(obj) {
-        let f = obj.getAttribute("for");
-        if (f) {
+        let labelAttribute = obj.getAttribute("for");
```
Comment (sanjaypradeep) [inline library/checkbox/checkbox.js]: What if it returns a `null or empty string or undefined`?

Validation would be helpful.

File: library/checkbox/checkbox.js [hunk 2]
```diff
@@ -7,21 +7,21 @@ export class Checkbox {
...
+        if (labelAttribute) {
           obj.addEventListener("click", function(event) {
-            let checkbox = document.getElementById(f);
+            let checkbox = document.getElementById(labelAttribute);
             if (checkbox) {
                 let state = checkbox.getAttribute("data-value");
-                state == "off" ? checkbox.setAttribute("data-value", "on") : checkbox.setAttribute("data-value", "off");
+                state === "off" ? checkbox.setAttribute("data-value", "on") : checkbox.setAttribute("data-value", "off");
```
Comment (sanjaypradeep) [inline library/checkbox/checkbox.js]: So values like `on` or `off` and other kinds, are hardcoded many places, can we put that in a separate `constant.js`

File: library/checkbox/checkbox.js (hunk 1 above)
Comment (alexmuzenhardt) [reply library/checkbox/checkbox.js]: We could add some of those validations. That won't hurt.
Could you open a new issue with that topic to track it? I will take that issue then and resolve it in the next days. Thanks.

File: library/checkbox/checkbox.js (hunk 2 above)
Comment (alexmuzenhardt) [reply library/checkbox/checkbox.js]: @javascriptteacher what do you mean?

File: library/checkbox/checkbox.js (hunk 1 above)
Comment (sanjaypradeep) [reply library/checkbox/checkbox.js]: @alexmuzenhardt yeah will create a new issue.

**Pull request merged**